governance/program/tests/process_create_mint_governance.rs

```diff
@@ -142,7 +142,7 @@ async fn test_create_mint_governance_without_transferring_mint_authority_with_au
             |i| {
```

Comment:
GH doesn't let me comment on `governance/program/tests/fixtures/spl_governance_voter_weight_addin.so`, so I'll do it here.  Right now, if you make a change to the addin, you need to build it and copy the so into here each time, right? Since the governance program and add-in are both in the same `BPF_OUT_DIR`, program-test should find them. You can have a nice error message saying `please run cargo build-bpf in governance/voter-weight-addin/program` if it isn't found